clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -743,6 +749,13 @@ static ParamDesc makeParamDesc(ASTContext &Ctx, const CXXBaseSpecifier &Src,
                          Ctx.getTrivialTypeSourceInfo(Ty));
 }
 
+// FIXME: Should we refactor makeParamDesc to just accept Name in all cases
```

Comment:
I'd probably think about having the 1st overload simply call this one instead.  Having the names auto-generated is nice, and would end up resulting in being called/named slightly differently from different places over time.